components/camel-spring-rabbitmq/src/main/java/org/apache/camel/component/springrabbit/SpringRabbitMQEndpoint.java

```diff
@@ -154,6 +154,10 @@
     @UriParam(defaultValue = "false", label = "advanced",
               description = "Sets whether synchronous processing should be strictly used")
     private boolean synchronous;
+    @UriParam(label = "consumer,advanced",
+              description = "Tell the broker how many messages to send in a single request. Often this can be set quite high to improve throughput."
+                            + " This param override the value from the component.")
```

Comment:
param -> parameter

But you do not have to tell this, its a common practice for Camel components where options can be configured as default on component level, and override per endpoint if needed.

Reply:
Thanks, will remove it